pages/api/stripe/checkout-session.js

```diff
@@ -0,0 +1,44 @@
+const { findOrCreateCustomer } = require('../../../lib/stripeHelpers')
+
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY)
+
+export default async (req, res) => {
+  if (req.method !== 'POST') return res.status(405).end()
+  const data = req.body
+
+  // Get or create customer
+  const customer = await findOrCreateCustomer({
+    name: `${data.firstName} ${data.lastName}`,
```

Comment:
i didn't require first and last name previously, so let's keep an eye on this to make sure it works as intended and customers who only have an email in our system (and no first / last name) can still login and see their donations